tensordict/tensordict.py

```diff
@@ -2692,6 +2692,23 @@ def get(
             # the regular get()
             return self._default_get(key, default)
 
+    def pop(
```

Comment:
Agree. I implemented pop() in subclasses separately because the codes of get() and del_() in subclasses might be different. For example, TensorDict.pop() supports nested key while LazyStackedTensorDict.pop() doesn't. Coding it in TensorDictBase might make others confused. However considering that behaviors of get() and del_() in subclasses finally will be similar, I agree that coding pop() in parent TensorDictBase class is a better idea.